[18.0][MIG] report_partner_address: Migration to 18.0 (#1113)
OCA/reporting-engine: report_partner_address/readme/USAGE.md

- Go to the partner record.
- Fill in Address Details, including translations for the active languages.

The address in reports and portal pages will use the value from Address Details.

If Address Details is left blank, the standard address format is used.

> Note: Address Details does not sync to child partners like Odoo’s standard address fields.

Reviewer: To be slightly more precise.
```suggestion
Note: Unlike standard address fields, Address Details is not inherited by child contacts.
```
